### DISCUSSION TITLE
GET_LEDGER_SUMMARY_VIEW_DATA report returning only until 4 January for multiple sellers

### DISCUSSION THREAD

**Discussion by abirafdirp** [amzn/selling-partner-api-models#3470] [General]
Hello All.

Per title, this issue appears with multiple sellers, tested in two different apps.
When we tried to download the report directly from seller central website, we have the same data returned.
No matter what the date range of the report request is, the latest will always be 4 January 2023.
We've tried many possible report options combination e.g. drop the aggregate, play with the date range, still 4 January 2023 is the latest returned.

Anyone else having this issue?

**Comment by ivanzonware** [comment]
Hello, 
I am experiencing the same but Seller Centrl Developer support is providing no answer to this query...
